ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.13.2__4.3.yaml

```diff
@@ -10,6 +10,9 @@ build_root:
     dockerfile_path: openshift/ci-operator/build-image/Dockerfile
 canonical_go_repository: knative.dev/serving
 images:
+- dockerfile_path: openshift/ci-operator/source-image/Dockerfile
```

Comment:
Curious: This Dockerfile is a noop. Can we just drop it potentially?

Reply:
No. A Dockerfile is really required by the ci-operator (and the OpenShift Build in the background)